### PR TITLE
[Camera] Add Camera() constructor for camera device internal selection

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Add Camera() constructor to select camera device automatically by CameraDeviceManager.

### API Changes ###

 - TCSACR-572

### CODE REVIEW

File: src/Tizen.Multimedia.Camera/Camera/Camera.cs [hunk 1]
```diff
@@ -69,12 +68,20 @@ public Camera(CameraDevice device)
         /// <summary>
         /// Initializes a new instance of the <see cref="Camera"/> class.
         /// </summary>
-        /// <remarks>CameraDevice and Type will be selected internally by CameraDeviceManager.</remarks>
-        /// <exception cref="InvalidOperationException">In case of any invalid operations.</exception>
+        /// <remarks>
+        /// If <see cref="CameraDeviceManager"/> is supported, <see cref="CameraDevice"/> will be selected automatically by internal policy.<br/>
+        /// (User can check whether <see cref="CameraDeviceManager"/> is supported or not by using <see cref="CameraDeviceManager.IsCameraDeviceManagerSupported"/>.)<br/>
```
Comment (wiertel) [inline src/Tizen.Multimedia.Camera/Camera/Camera.cs:73]: I would make a link (with `<see `) only for the first instance of the API item in the text, but it's not an error. No need to change anything here.

**Review by wiertel — APPROVED** [review]

**Review by wiertel — CHANGES_REQUESTED** [review]
API level 11 or 12?

File: src/Tizen.Multimedia.Camera/Camera/CameraEnums.cs [hunk 2]
```diff
@@ -26,7 +26,7 @@ namespace Tizen.Multimedia
     public enum CameraDevice
     {
         /// <summary>
-        /// The CameraDevice will be decided internally by target policy.
+        /// The CameraDevice will be decided automatically by internal policy, if CameraDeviceManager is supported.
```
[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Multimedia.Camera/Camera/CameraEnums.cs:29]: Actually, the description of Default is for internal developer, not for public user.

File: src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs [hunk 3]
```diff
@@ -65,6 +65,28 @@ public CameraDeviceManager()
                                         d.Type == CameraDeviceType.Network)
                             .Any();
 
+        /// <summary>
+        /// Gets the camera device manager state.
```
[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs:69]: Okay I'll change it as your suggestion.
`Gets the state of Camera Device Manager support`

File: src/Tizen.Multimedia.Camera/Camera/CameraEnums.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Multimedia.Camera/Camera/CameraEnums.cs:29]: Actually, the description of Default is for internal developer, not for public user.

File: src/Tizen.Multimedia.Camera/Camera/Camera.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Camera/Camera/Camera.cs:73]: I agree with you. I'll remove duplicated link after first one.

**Comment by hsgwon** [comment]
> API level 11 or 12?

API level was wrong. I fixed it as API12.

**Pull request closed without merging**